### PR TITLE
Fix install 3rd app

### PR DESCRIPTION
**Pull request by duanxianbo**
[(task:20640)](https://staging.teamyapp.com/teams/107/tasks/20640)

### CODE REVIEW

File: src/storage/graph/app.node.ts [hunk 1]
```diff
@@ -207,7 +207,6 @@ export class AppNode implements App {
 
     public isInstalled(teamId: number): boolean {
         const appState = this.localStore.getState();
```
Comment (duanxianbo) [inline src/storage/graph/app.node.ts:209]:
```suggestion
        const appState = this.localStore.getState();
        
```

File: src/storage/syncer/stateMutators.ts [hunk 2]
```diff
@@ -943,7 +943,7 @@ export function mergeAppVersion(
         appId: appId,
         createdAt: toDate(version.createdAt) || localOldVersion.createdAt,
         creatorUserId:
-            toInt(version.createdBy.id) || localOldVersion.creatorUserId,
+            toInt(version.createdBy?.id) || localOldVersion.creatorUserId,
```
Comment (naturetime10) [inline src/storage/syncer/stateMutators.ts]: Can you explain what happened?

Comment (naturetime10) [inline src/storage/syncer/stateMutators.ts]:
```suggestion
            version.createdBy !== undefined ? toInt(version.createdBy!.id) || localOldVersion.creatorUserId,
```

Comment (duanxianbo) [reply src/storage/syncer/stateMutators.ts]: when we install app to the the team, we return app with app version, and do merge app and merge app version, but in the returned app version, we do not have createdBy

**Review by naturetime10 — APPROVED** [review]

File: src/storage/syncer/stateMutators.ts (hunk 2 above)
Comment (naturetime10) [reply src/storage/syncer/stateMutators.ts]: @duanxianbo Should we return `createdBy` instead?

File: src/storage/syncer/stateMutators.ts [hunk 3]
```diff
@@ -943,7 +943,7 @@ export function mergeAppVersion(
         appId: appId,
         createdAt: toDate(version.createdAt) || localOldVersion.createdAt,
         creatorUserId:
-            toInt(version.createdBy.id) || localOldVersion.creatorUserId,
+            version.createdBy !== undefined ? toInt(version.createdBy!.id) || localOldVersion.creatorUserId,
```
Comment (naturetime10) [inline src/storage/syncer/stateMutators.ts]:
```suggestion
            version.createdBy !== undefined ? toInt(version.createdBy!.id) : localOldVersion.creatorUserId,
```

**Review by naturetime10 — APPROVED** [review]

**Pull request merged**